### PR TITLE
Feature/checkoutform

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Committing the changes for checkout form story.Please review and merge.

### CODE REVIEW

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java [hunk 1]
```diff
@@ -16,7 +16,11 @@
 	String updateCartItemPath = "/us/V1/guest-carts/{cartId}/items/{ItemId}";
 	String emptyCartPah = "/us/V1/guest-carts";
 	String guestCartItemDeletePah = "/us/V1/guest-carts/{cartId}/items/{itemId}";
+
     String servicePathCartUpdateItems = "/V1/guest-carts/";
+    String guestCartShippingInformation = "/us/V1/guest-carts/{cartId}/shipping-information";
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java]: @nitheshkumar-hcl  append Path in variable name.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/.content.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="This is Check Out Form"
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/.content.xml]: @nitheshkumar-hcl  update more details in description.

File: core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+/**
+ * Servlet that delete the existing cart item by hitting the Magento service.
+ */
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java]: @nitheshkumar-hcl  update the servlet class description

File: core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+						responseObject.put("message", "something went wrong while Order Create from cart.");
+					}
+				}
+			} else {
+				responseObject.put("message", "Missing Parameter in payload.");
+				responseObject.put("status", Boolean.FALSE);
+			}
+			response.getWriter().print(responseObject);
+		} catch (Exception e) {
+			LOG.info("Error Occured while executing delete item from cart. Full Error={} ", e);
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java]: @nitheshkumar-hcl  update looger to error and description

File: core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java (hunk 1 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java]: variable name is updated and committed the changes

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/.content.xml (hunk 2 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/.content.xml]: updated the details in  description

File: core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java (hunk 3 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java]: updated the  servlet class description

File: core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java (hunk 4 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java]: Updated the logger to error and description

**Pull request merged**